gym/vector/utils/__init__.py

```diff
@@ -6,7 +6,7 @@
     read_from_shared_memory,
     write_to_shared_memory,
 )
-from gym.vector.utils.spaces import BaseGymSpaces, _BaseGymSpaces, batch_space, iterate
```

Comment:
I kept this export deliberately. It was a bad idea and shouldn't be made private initially, but it's not actively harmful. One less breaking change for people to complain about.

Reply:
This is in `vector/utils/__init__.py` so if someone is getting this from here then there is an issue